base_miner/gating_mechanisms/face_gate.py

```diff
@@ -24,128 +24,27 @@ def __init__(self, gate_name: str = 'FaceGate', predictor_path=DLIB_FACE_PREDICT
         self.face_predictor = dlib.shape_predictor(predictor_path)
         super().__init__(gate_name, "face")
 
-    def get_keypts(self, image, face):
```

Comment:
Thanks for moving to a utils file, these preprocessing functions shouldn't live in the gate implementation.

Small suggestion: Create a `utils/` subdir in `gating_mechanisms/ `